<launch>

    <arg name="config_path" default = "$(find feature_tracker)/../config/3dm/3dm_config.yaml" />
	<arg name="vins_path" default = "$(find feature_tracker)/../config/../" />

    <node name="feature_tracker" pkg="feature_tracker" type="feature_tracker" output="log">
        <param name="config_file" type="string" value="$(arg config_path)" />
        <param name="vins_folder" type="string" value="$(arg vins_path)" />
    </node>
    <node name="slam_estimator" pkg="slam_estimator" type="slam_estimator" output="screen">
        <param name="config_file" type="string" value="$(arg config_path)" />
        <param name="vins_folder" type="string" value="$(arg vins_path)" />
    </node>
     
</launch>
